python/scripts/genetic_search.py

```diff
@@ -95,6 +95,7 @@ def genetic_search(
         candidate_generator=candidate_generator,
         cost_function=cost_function,  # TODO: parameterize when there are more cost functions
         max_iterations=max_iterations,
+        cache_size=50
```

Comment:
Can we set this as a command line arg and put the default in the config :)

Reply:
A very unreasonable request indeed